backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -134,7 +137,7 @@ public static App getInstance() {
     return app;
   }
 
-  public static boolean migrate(Config config)
+  public static boolean migrate(DatabaseConfig databaseConfig, List<MigrationConfig> migrations)
```

Comment:
This class changed to support test database config and initialize DAOs for test database